### DISCUSSION TITLE
请问为什么dataconfig 配置的字段顺序 在渲染后 顺序错位了

### DISCUSSION THREAD

**Discussion by ornncrisol** [antvis/S2#3002] [Q&A]
![微信图片_20241129142221](https://github.com/user-attachments/assets/898ac8b1-10a2-425c-8669-1dfcf1d6ae25)
![微信图片_20241129142233](https://github.com/user-attachments/assets/21e49778-76bb-4f17-a697-5f2638b5757c)
![微信图片_20241129142246](https://github.com/user-attachments/assets/4f2290d0-f409-4cb2-87c9-ae04666d544a)

dfhsl这个字段应该在values的第一位

**Comment by lijinke666** [comment]
请提供在线的最小可复现 codesandbox demo, 只有图片看不出为什么 https://github.com/antvis/S2/issues/1904
